packages/rpc-types/src/typed-numbers.ts

```diff
@@ -1,24 +1,12 @@
-/**
- * FIXME(solana-labs/solana/issues/30341) Beware that any value above 9007199254740991 may be
- * truncated or rounded because of a downcast to JavaScript `number` between your calling code and
- * the JSON-RPC transport.
- */
-export type U64UnsafeBeyond2Pow53Minus1 = bigint;
+export type U64 = bigint;
+export type I64 = bigint;
```

Comment:
I don't have a strong opinion here, but do we need these aliases?

Reply:
I also don't have strong opinions here. I guess it's nice to mark that a certain bigint is using 8 bytes max and is signed/unsigned. Happy to add another PR if we decide to dissolve them though.

Reply:
I'd lean toward removing them too, but also not a strong opinion. We're not using them for the relevant codecs and I don't think we should, which makes me think they're probably not adding much.

Reply:
There's marking to be helpful, but on the flipside there's trusting that the type enforces the runtime behaviour, which here it does not (ie. these could, in fact, be negative and TypeScript would not complain).

Reply:
Sounds like we're all leaning toward dissolving them. Follow up PR incoming.

Reply:
https://github.com/solana-labs/solana-web3.js/pull/3467